.github/MIGRATION.md

```diff
@@ -14,6 +20,131 @@
     - [Using blockstack.js](#using-blockstackjs-2)
     - [Using @stacks/encryption or @stacks/auth](#using-stacksencryption-or-stacksauth)
 
+## Stacks.js (&gt;=5.x.x) → (7.x.x)
+
+### Breaking Changes
+
+- The `@stacks/network` `new StacksNetwork()` objects were removed. Instead `@stacks/network` now exports objects like `STACKS_MAINNET`, which are static (and shouldn't be changed for most use-cases). [Read more...](#stacks-network)
```

Comment:
I'd would enumerate the objects here (otherwise the plural makes it hard to understand)

Reply:
👍 will do

Reply:
Added them (not mocknet, to discourage use) 😉